### ISSUE TITLE
video resolution in metadata after apply scale avfilter

### ISSUE THREAD

**Issue by sherpya** [wang-bin/QtAV#536]
Hi, if I apply avfilter "scale", the metadata still contain the original resolution, there is a way to get the scaled resolution instead?

I'm using
```
	m_video_filter.setOptions("scale=320:240");
	m_video_filter.setEnabled(true);
	//m_video_player->uninstallFilter(&m_video_filter);
	m_video_player->installVideoFilter(&m_video_filter);
```

btw:
- do I/may I call uninstallFilter before installVideoFilter()?
- installVideoFilter() is deprecated, what is the non deprecated way to add a filter?

Thanks

**Comment by wang-bin** [comment]
what metadata? how do you get it?

* yes
* installFilter()

**Comment by sherpya** [comment]
thanks for the reply, `metaData.resolution` property of `QML AVPlayer`

**Comment by wang-bin** [comment]
`metaData.resolution` is the property of video stream. It does not change. Imagine there are 2 VideoOutput for the same player, each 1 has a scale filter(though it's not supported now), then what should `player.metaData.resolution` be?
What do you want to do? Maybe there are other solutions.

**Comment by sherpya** [comment]
yes ffmpeg does not changes the stream resolution when applying a filter, I just wondering if there is some way to get the final resolution

**Comment by wang-bin** [comment]
try the latest code. I add `frameSize` property to VideoOutput
